packages/core/src/components/table/table-body-row-expandable/table-body-row-expandable.tsx

```diff
@@ -32,6 +35,15 @@ export class TdsTableBodyRowExpandable {
    *  Take in mind that expandable control is column too */
   @Prop() colSpan: number = null;
 
+  /** Sets default expanded value of row */
+
+  /** ID for the table row. Randomly generated if not specified. */
+  @Prop({ reflect: true }) rowId: string = generateUniqueId();
+
+  /** Sets isExpanded state to true or false externally */
+  @Prop({ reflect: true }) expanded: boolean = false;
+
```

Comment:
```suggestion
  /** Sets isExpanded state to true or false externally */
  @Prop({ reflect: true }) expanded: boolean;
```
Related to connectedCallback later on, should we keep this undefined then?

Reply:
Yes, that makes sense